kolibri/plugins/setup_wizard/assets/src/views/onboarding-forms/DefaultLanguageForm.vue

```diff
@@ -24,8 +24,6 @@
     inject: ['wizardService'],
     methods: {
       handleSubmit() {
-        const currentLanguageId = this.$store.state.onboardingData.language_id;
-        this.$store.commit('SET_LANGUAGE', currentLanguageId);
         this.wizardService.send('CONTINUE');
```

Comment:
can you remind me where the language setting is handled now? is it just in the pop up that is outside the scope of the flow?

Reply:
The `LanguageSwitcherList` makes the call to the backend to set the default language. This was only stored in Vuex for the frontend's reference previously I believe